Java/app/src/main/java/com/taboola/sdk3example/sdk_native/SDKNativeMenu.java

```diff
@@ -0,0 +1,104 @@
+package com.taboola.sdk3example.sdk_native;
+
+import android.os.Bundle;
+
+import android.view.View;
+import android.view.Window;
+import android.view.WindowManager;
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentTransaction;
+
+import com.taboola.sdk3example.R;
+
+
+public class SDKNativeMenu extends AppCompatActivity   implements SDKNativeMenuFragment.OnFragmentInteractionListener {
+
+
+
+    private Toolbar mToolbar;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        requestWindowFeature(Window.FEATURE_NO_TITLE);
+        this.getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,WindowManager.LayoutParams.FLAG_FULLSCREEN);
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.sdk_native_menu);
+
+
+        mToolbar = findViewById(R.id.toolbar);
+        setSupportActionBar(mToolbar);
+        getSupportActionBar().setLogo(R.drawable.ic_taboola);
```

Comment:
no need to getSupportActionBar twice, also, I think you can just use mToolBar

Reply:
Done